pkg/ovsdb/monitor.go

```diff
@@ -24,7 +25,7 @@ const (
 )
 
 type updater struct {
-	Columns          map[string]bool
+	Columns          map[string]*libovsdb.ColumnSchema
```

Comment:
Basically if you hold the TablesSchema you get the exact functionality you need by using `LookupColumn()` w/o the need to create/maintain the Columns field

Reply:
I changed the monitor internals, and removed the map at all.